scripts/update_leaderboard.py

```diff
@@ -55,7 +55,7 @@ def get_name_from_id(user_id: str) -> str:
                     {% for submission in problem.submissions %}
                     <div class="submission{% if submission.rank == 1 %} first{% elif submission.rank == 2 %} second{% elif submission.rank == 3 %} third{% endif %}"
                          data-user="{{ submission.user }}"
-                         data-time="{{ submission.time }}s"
+                         data-time="{{ submission.time }}μs"
```

Comment:
are these µs or ns? I think we generally use 10^9 as conversion factor, which would mean ns.

Reply:
cc @alexzhang13 who first made this change